src/Notifications/NotificationCenterController.cs

```diff
@@ -64,8 +69,11 @@ private void WebView_CoreWebView2InitializationCompleted(object sender, Microsof
                 htmlString = htmlString.Replace("mainJs", jsString);
             }
 
-            if(notificationUIPopup.webView.CoreWebView2 != null)
+            if (notificationUIPopup.webView.CoreWebView2 != null)
+            {
                 notificationUIPopup.webView.CoreWebView2.NavigateToString(htmlString);
+                RefreshNotifications();
```

Comment:
Just want to be clear, this currently does not work immediately as you described?

Reply:
Nope, after the meeting @filipeotero helped in getting it to work, so it does re-renders now.

Reply:
Thank you both for looking into this and getting it to work 👍🏻